<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-0EC05GSJ" ignored="2" name="Dependency Test" tests="2" failures="1" timestamp="2021-04-07T13:02:43 IST" time="0.085" errors="0">
  <testcase name="doUserReg" time="0.001" classname="testcases.TestCase3">
    <failure type="java.lang.AssertionError" message="User not registered Successfully">
      <![CDATA[java.lang.AssertionError: User not registered Successfully
at testcases.TestCase3.doUserReg(TestCase3.java:10)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- doUserReg -->
  <testcase name="doLogin" time="0.0" classname="testcases.TestCase3">
    <skipped/>
  </testcase> <!-- doLogin -->
  <testcase name="thirdTest" time="0.0" classname="testcases.TestCase3">
    <ignored/>
  </testcase> <!-- thirdTest -->
  <testcase name="fourthTest" time="0.0" classname="testcases.TestCase3">
    <ignored/>
  </testcase> <!-- fourthTest -->
</testsuite> <!-- Dependency Test -->
